pkg/statdb/statdb.go

```diff
@@ -52,4 +53,5 @@ type NodeStats struct {
 	UptimeRatio        float64
 	UptimeSuccessCount int64
 	UptimeCount        int64
+	Meta               pb.NodeOperator
```

Comment:
Can we have this field named `Operator` instead of `Meta`? "Meta" is too generic. When referenced in code `stats.Operator.Wallet` looks clearer than `stats.Meta.Wallet`.